.github/workflows/publish-and-deploy.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
   push_to_registry:
     name: Push Docker Image to Docker Hub
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Comment:
Good catch, though this means it will be one more thing to comment out when we're testing changes in dev (at least until gamma is a thing)